fieldservice/models/fsm_order.py

```diff
@@ -157,7 +157,7 @@ def _onchange_location_id_customer(self):
     # Planning
     person_id = fields.Many2one("fsm.person", string="Assigned To", index=True)
     person_phone = fields.Char(related="person_id.phone", string="Worker Phone")
-    scheduled_date_start = fields.Datetime(string="Scheduled Start (ETA)")
+    scheduled_date_start = fields.Datetime(string="Scheduled Start (ETA)", default=fields.Datetime.now())
```

Comment:
Wither use a lambda, or pass the function reference
```suggestion
    scheduled_date_start = fields.Datetime(string="Scheduled Start (ETA)", default=fields.Datetime.now)
```